### DISCUSSION TITLE
Communication between multiple domain.

### DISCUSSION THREAD

**Discussion by mahesh2392** [OpenDDS/OpenDDS#3954] [Q&A]
I have application A ,B, and C, A can only communicate(write/read) with B and C only communicate(read/write) with B. At the end in B publish some data which only received by B and then B will published same data which will be received only by A. 
What I am doing as data is same I am using same topic names I am just creating two domain  [A B Domain 1]   [B C Domain 2] I am creating separate participant and creating writers/readers with different domain IDs e.g. for   [B C Domain 2]  i am creating writers/readers with domain id 2 and for  [A B Domain 1] writers/readers with domain id 1 B is act as mediator of two domain but what i observe communication happening between only one domain at  time either [A B Domain 1] or  [B C Domain 2] depend on which application start first if I miss any configuration related to SD please mention it. currently I am testing it on single linux machine with three terminal.
Thanks 
Mahesh

**Comment by mitza-oci** [comment]
Please show us the details of how you're configuring OpenDDS -- both source code and .ini file, if you're using one.

I'm not sure what the phrase "configuration related to SD" means.
